tests/js/spec/views/organizationIntegrations/integrationRepos.spec.jsx

```diff
@@ -15,6 +15,92 @@ describe('IntegrationRepos', function() {
     const integration = TestStubs.GitHubIntegration();
     const routerContext = TestStubs.routerContext();
 
+    describe('successful save', async function() {
+      let addRepo = Client.addMockResponse({
+        url: `/organizations/${org.slug}/repos/`,
+        method: 'POST',
+        body: {
+          id: 9,
+          integrationId: '1',
+          name: 'getsentry/sentry',
+          url: 'https://github.com/getsentry/sentry',
+          provider: {
+            name: 'GitHub',
```

Comment:
So why do we use GitHub here and not the Example integration as we would in the django tests? Seems like this is a broad change that isn't github specific.

Reply:
I was matching up with the `TestStubs.GitHubIntegration()` which is the test fixture for integration objects.

Reply:
@lauryndbrown there was already a `JiraIntegration` but needed one to test repository functionality so I add `GitHubIntegration`. We could potentially have a sample integration test stub but generalizing for all the integrations seems like more effort than it was worth at the time.